WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Amount/SimplePaymentsAmount.swift

```diff
@@ -139,9 +139,10 @@ struct SimplePaymentsAmount: View {
                     dismiss()
                     viewModel.userDidCancelFlow()
                 })
-                    .disabled(viewModel.disableCancel)
+                    .disabled(viewModel.disableViewActions)
```

Comment:
Nit: This isn't needed now because everything (including the cancel button) is disabled a few lines below, right?

Reply:
Oh yeah! We don't need this anymore!